jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/MojoCommon.java

```diff
@@ -43,22 +44,21 @@ static boolean isWarProject(MavenProject project) {
   }
 
   /**
-   * Gets the extra directory path from a {@link JibPluginConfiguration}. Returns {@code (project
-   * dir)/src/main/jib} if null.
+   * Gets the list of extra directory paths from a {@link JibPluginConfiguration}. Returns {@code
+   * (project dir)/src/main/jib} by default if not configured.
    *
    * @param jibPluginConfiguration the build configuration
-   * @return the resolved extra directory
+   * @return the list of resolved extra directories
    */
-  static Path getExtraDirectoryPath(JibPluginConfiguration jibPluginConfiguration) {
-    return jibPluginConfiguration
-        .getExtraDirectoryPath()
-        .orElse(
-            Preconditions.checkNotNull(jibPluginConfiguration.getProject())
-                .getBasedir()
-                .toPath()
-                .resolve("src")
-                .resolve("main")
-                .resolve("jib"));
+  static List<Path> getExtraDirectoryPaths(JibPluginConfiguration jibPluginConfiguration) {
```

Comment:
getExtraDirectories? Is the return type should be sufficient for us to understand it is `java.nio.Path` ?